Editor/AGS.Editor/Utils/Validation.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace AGS.Editor.Utils
+{
+    public class Validation
+    {
+        public static bool FilenameIsValid(string filename)
+        {
+            return !(filename.IndexOfAny(Path.GetInvalidFileNameChars()) >= 0);
+        }
+
+        public static bool PathDoesNotExist(string path)
```

Comment:
Usually public bool api names is phrased as a positive question, like you do everywhere else, so it's a little strange to see it here. It's a nitpick though

Reply:
It is a positive result from the perspective of an action on project files, not checking that something exists (because files and directories are checked), but more of a "is this filesystem path available for use?". But I see what you mean, I'll rename it.